graphql/pupil/mutations.ts

```diff
@@ -184,6 +190,13 @@ export async function updatePupil(
         throw new PrerequisiteError(`Only Admins may change the email without verification`);
     }
 
+    let dbSchool: School | undefined;
+    try {
+        dbSchool = await findOrCreateSchool(school);
```

Comment:
Good catch, they can do it via `meUpdate`.

Reply:
I included similar validations for students now that I know I didn't add them.

Reply:
You added all but school? :)

Reply:
Oh, that was intentional, for now, we don't offer pupils the UI to change their school, but we probably will.
Also after https://github.com/corona-school/project-user/issues/1387 they'll use this to change the existing `schoolType`/`state` fields.

Reply:
So we want to allow pupils to create school entries? That sounds like it'll create a mess longterm (typo, etc.)

Reply:
Hmm, they're actually already able to do it during registration, of course, it's just once per user.

So, maybe it would be safer to block pupils from updating schools using this function and then we could create a separate mutation specifically for updating schools, with strict rate limits 🤔
What do you think?

Reply:
Separate mutation sounds good - not sure if it needs a rate limit. Maybe it is enough to make it hard in the UI to create a new school ("did you mean <without typo> instead?")

Reply:
>Maybe it is enough to make it hard in the UI to create a new school

Alright, when we build a UI for updating schools we'll keep that in mind